facefusion/vision.py

```diff
@@ -228,6 +234,18 @@ def equalize_frame_color(source_vision_frame : VisionFrame, target_vision_frame
 	return target_vision_frame
 
 
+def calc_histogram_difference(source_vision_frame : VisionFrame, target_vision_frame : VisionFrame) -> float:
```

Comment:
1. use this in the unit test for match_frame_color
2. add a stand alone unit testing for this method